### ISSUE TITLE
Suggest widget layout should close when you press Ctrl + S.

### ISSUE THREAD

**Issue by Tekbr** [Microsoft/vscode#26488]
- VSCode Version: 1.13.0 - Insiders
- OS Version: Windows 10

-- Sorry for the English, I used Google Translator --

Suggest widget layout should close when you press <kbd>Ctrl</kbd> + <kbd>S</kbd>.

**Comment by jrieken** [comment]
@Tekbr Can you justify why that should be done?

**Comment by Tekbr** [comment]
@jrieken Of course.

This is a simple example in html file. (In actual situation the document has many lines).
The situation holds true for other types of files.

Have the following sentences. (You can copy and paste).

```
<h5> VSCode-Insiders </h5>

<h5> VSCode-Stable </h5>
```


Now enter the following phrase.

`<h5> I like VSCode </h5>`

Press <kbd>Ctrl</kbd> + <kbd>S</kbd>.

See I want to type just this: "I like VSCode", nothing more. Now I need to scroll to lines above or below.

I get stuck in a loop. I would press <kbd>Esc</kbd> to be able to exit this loop.

Another one if I pressed <kbd>Ctrl</kbd> + <kbd>S</kbd>, most likely this suggest widget does not interest me anymore. And if I wanted it again just hit <kbd>Ctrl</kbd> + <kbd>space</kbd>.

**Comment by samuela** [comment]
I'd like to echo this. As I wrote in #38478 

> IntelliSense is great, albeit pretty eager at times. It would be great if it automatically hid once the user saves the file. Presumably a file save corresponds to the user having just written whatever they wanted to anyways so there's really no need to keep the IntelliSense popup hanging around. I find myself doing this cmd-s, esc key combination constantly in VSCode.
>
> I've noticed that Atom implements this feature and while I didn't even think about it being there at the time, now I sorely miss it.

Overall, Atom just feels more polished to me than VSCode. It may be slightly less featureful, but overall it really *feels* like attention and care have been given to these details. Another issue like this is https://github.com/Microsoft/vscode/issues/38471. I'd like to switch to VSCode but even these minor things add up to serious annoyances when they accumulate over hours and days of coding.

**Comment by samuela** [comment]
Is there any hope in getting this taken care of? Would the team be open to PRs?

**Comment by Tekbr** [comment]
-- Sorry for the English, I used Google Translator --

@jrieken , you put this issue out of scope, but you did not pay attention after the last response, if you were able to reproduce the problem.

I ask you to reopen the case ... it is a very annoying situation. At Atom it works fine.

As I said above, if I saved the document it's because I do not need the suggestion any more. I believe you have already stepped into this situation.

Maybe he would put the case as "good first issue." It may be that someone else in the community is interested in solving the problem (I do not know if it would be @samuela  case)

> Is there any hope in getting this taken care of? Would the team be open to PRs?
